src/backend/access/appendonly/appendonlywriter.c

```diff
@@ -923,30 +923,34 @@ usedByConcurrentTransaction(AOSegfileStatus *segfilestat)
  */
 int
 addCandidateSegno(AOSegfileStatus **maxSegno4Seg, int segment_num, AOSegfileStatus *candidate_status, bool isKeepHash)
-{  
+{
+	int remaining_num  = segment_num;
```

Comment:
The space of this chunk of code is messy.
`{}` placement is not consistent, even in a single function.